README.md

```diff
@@ -692,26 +692,16 @@ export const getConfig = async () => {
 
 ```css
 /* ./src/styles.css */
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-```
-
-```js
-// ./tailwind.config.js
-export default {
-  content: ['./src/**/*.{js,jsx,ts,tsx}'],
-};
+@import "tailwindcss";
```

Comment:
Would you mind to change the `styles.css` files in the `examples` folder with this change?

Reply:
Isn't it what we have in the current examples?

Reply:
oh, you're right.  My mistake. Please disregard this.